easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023b.yml

```
@@ -8,3 +8,9 @@ easyconfigs:
   - matplotlib-3.8.2-gfbf-2023b.eb:
       options:
         from-pr: 19552
+  - DP3-6.0-foss-2023b.eb:
```

So we probably need this to fix the `casacore` build:
```suggestion
  - casacore-3.5.0-foss-2023b.eb:
      options:
        from:pr: 19840
        include-easyblocks-from-pr: 3088
  - DP3-6.0-foss-2023b.eb:
```

---

Thanks! I've added `include-easyblocks-from-pr: 3088` to all the software and dependencies that had an issue with the Python path.

---

Bob's suggestion had a typo: `from:pr:` instead of `from-pr:` (the latter is correct)